modules/database/src/main/java/org/eclipse/xpanse/modules/database/serviceconfiguration/update/ServiceConfigurationUpdateRequest.java

```diff
@@ -35,8 +41,23 @@ public class ServiceConfigurationUpdateRequest implements Serializable {
     private static final long serialVersionUID = 8759112725757851274L;
 
     @Id
-    @Column(name = "ORDER_ID", nullable = false)
-    private UUID orderId;
+    @GeneratedValue(strategy = GenerationType.UUID)
+    @Column(name = "ID", nullable = false)
+    private UUID id;
+
+    @ManyToOne
+    @JoinColumn(name = "ORDER_ID", nullable = false)
+    private ServiceOrderEntity serviceOrderEntity;
+
+    @ManyToOne
+    @JoinColumn(name = "SERVICE_ID", nullable = false)
+    private DeployServiceEntity deployServiceEntity;
```

Comment:
Will these records be deleted by cascading when associated services are destroyed and purged from the database

Reply:
yes, please check that all child tables are also deleted automatically when the service is purged.